src/content/resources/concepts/layer/control/catalog/_index.md

```diff
@@ -17,7 +17,7 @@ suppresstopiclist: true
 
 | Catalog Schema | Catalog Converters | Reference
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/json-index/) |
 | [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_catalog_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/catalog/xml-index/) |
```

Comment:
The link https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml is broken. Needs to be replaced.

Reply:
I figured out how I missed this, I rebased the half of links that were incorrect (I completed the XML to JSON link replacement, but not the JSON to XML ones you cited). I rebased the original commit in the series and this was addressed.